host/src/FunctionsNetHost/Grpc/IncomingGrpcMessageHandler.cs

```diff
@@ -10,10 +10,13 @@ internal sealed class IncomingGrpcMessageHandler
     {
         private bool _specializationDone;
         private readonly AppLoader _appLoader;
+        private readonly GrpcWorkerStartupOptions _grpcWorkerStartupOptions;
+        private const string HostEndpoint = "Functions:Worker:HostEndpoint";
```

Comment:
We have a class `EnvironmentVariables` to store environment variable keys. Let's move this there.